script/download-standalone.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -eu
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)" # Figure out where the script is running
+. "$SCRIPT_DIR"/ci/lib/robust-bash.sh
+. "$SCRIPT_DIR"/ci/lib/utils.sh
+
+require_binary curl
+require_binary unzip
+
+STANDALONE_VERSION=$(cat ./standalone/install.ts | grep "PACT_STANDALONE_VERSION = '" | egrep -o "([0-9][\.0-9]+[0-9])")
+BASEURL=https://github.com/pact-foundation/pact-ruby-standalone/releases/download
+STANDALONE_DIR="$SCRIPT_DIR/../standalone"
+
+warn "Cleaning standalone directory $STANDALONE_DIR"
+rm -rf $STANDALONE_DIR/*${STANDALONE_VERSION}*
+
+function download_standalone {
+  if [ -z "${1:-}" ]; then
+    error "${FUNCNAME[0]} requires the environment filename suffix"
+    exit 1
+  fi
+  STANDALONE_FILENAME="$2"
+
+  URL="${BASEURL}/v${STANDALONE_VERSION}/${1}"
+  DOWNLOAD_LOCATION="$STANDALONE_DIR/${STANDALONE_FILENAME}"
+  echo $URL
+
+  log "Downloading verifier $STANDALONE_VERSION to $DOWNLOAD_LOCATION"
+  download_to "$URL" "$DOWNLOAD_LOCATION"
+  if [ "${STANDALONE_FILENAME%zip}" != "${STANDALONE_FILENAME}" ]; then
+    unzip "$DOWNLOAD_LOCATION" -d "${DOWNLOAD_LOCATION%.*}"
```

Comment:
If we extract these when we download them, I think the npm package will be much larger. I was thinking it'd be better to extract them when they're needed instead

Reply:
(I think the ffi doesn't matter as much, because it's only one file, and it's smaller. Although maybe it would be better to only extract as needed in general)

Reply:
The npm package is a tarball, so it will be compressed anyway.

Reply:
Just did a `--dry-run` locally to confirm this:

```
npm notice === Tarball Details ===
npm notice name:          @pact-foundation/pact-core
npm notice version:       12.0.0
npm notice package size:  29.9 MB
npm notice unpacked size: 87.0 MB
npm notice shasum:        cc0fb615407c4f6d0f0580196816e67877beb325
npm notice integrity:     sha512-zY8qGF8Qpg207[...]nMb1mVcRf7mIA==
npm notice total files:   2419
npm notice
```

You can see that the package is compressed, so no need to double compress it.